lib/playwright/channel_owners/browser_context.rb

```diff
@@ -171,14 +171,6 @@ def new_page(&block)
     end
 
     def cookies(urls: nil)
-      target_urls =
-        if urls.nil?
-          []
-        elsif urls.is_a?(Enumerable)
-          urls
-        else
-          [urls]
-        end
```

Comment:
Thank you :)

This is my fault that target_url must be set to send_message_to_server.
https://github.com/YusukeIwaki/playwright-ruby-client/pull/209/